### ISSUE TITLE
Can't install with engine-strict

### ISSUE THREAD

**Issue by felixfbecker** [storybooks/storybook#3029]
Runnning `npm install @storybook/react --engine-strict` (or with `engine-strict` in .npmrc) with npm 5 fails with

```
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for podda@1.2.2: wanted: {"npm":"^3.0.0"} (current: {"node":"8.9.1","npm":"5.6.0"})
npm ERR! notsup Not compatible with your version of node/npm: podda@1.2.2
npm ERR! notsup Not compatible with your version of node/npm: podda@1.2.2
npm ERR! notsup Required: {"npm":"^3.0.0"}
npm ERR! notsup Actual:   {"npm":"5.6.0","node":"8.9.1"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/felix/.npm/_logs/2018-02-19T22_21_14_142Z-debug.log
```
Afaik yarn has this on by default too.

Dependency tree:

```
sourcegraph@0.0.1 /Users/felix/go/src/sourcegraph.com/sourcegraph/sourcegraph/web
└─┬ @storybook/react@3.3.13
  └─┬ @storybook/ui@3.3.13
    └── podda@1.2.2
```

Tracked in https://github.com/arunoda/podda/issues/6, but obviously severely impacting storybook.
PR has been open since August: https://github.com/arunoda/podda/pull/5

**Comment by Hypnosphi** [comment]
Related: #2313

**Comment by Hypnosphi** [comment]
I'd base on ideas from https://github.com/storybooks/storybook/pull/1334

**Comment by Hypnosphi** [comment]
Released [3.3.14](https://github.com/storybooks/storybook/releases/tag/v3.3.14)
